README.md

```diff
@@ -136,6 +137,7 @@ wsa://com.android.settings
 | Genshin Impact (Cloud app) || ✅ |||
 | Geekbench |5.4.1| ✅ |||
 | GeoGebra | 5.0.674.0 | ✅ | | |
+| Girls' Frontline (EN) | 2.0702_362 (362) | ⚠️ || Unconfirmed: Game will always perma freezes after the plash screen, already tried restarting the PC but doesn't work. After installing WSL2, game can now run. AMD system PC |
```

Comment:
```suggestion
| Girls' Frontline (EN) | 2.0702_362 (362) | ⚠️ || Game freezes after splash screen, works after installing WSL2  |
```

Can you please clarify what exactly do you mean when you say installing WSL2? Is it the same as any other WSL deployment using ``wsl --install``, or have you tried something extra?

Reply:
Yes, sorry for that, I only use `wsl --install` and that is, after I installed it yesterday, the game actually runs today, so I'm a little confused